client/components/jetpack/wpcom-business-at/index.tsx

```diff
@@ -36,6 +36,11 @@ import JetpackScanSVG from 'assets/images/illustrations/jetpack-scan.svg';
 
 interface Props {
 	product: 'backup' | 'scan';
+	automatedTransferStatus: {
+		eligibility: EligibilityData;
+		fetchingStatus: boolean;
+		status: string;
```

Comment:
`status` can be one of the values of `transferStates`. Is there a way to set a type from an object's values? I would like to write something like `status: Object.values(transferStates);` instead of `status: string;`.

Reply:
Maybe try [`keyof`](https://mariusschulz.com/blog/keyof-and-lookup-types-in-typescript)?